components/cam/src/physics/cam/restart_physics.F90

```diff
@@ -308,7 +315,72 @@ subroutine write_restart_physics (File, cam_in, cam_out, pbuf2d, phys_state)
          
          call pio_write_darray(File, trefmxav_desc, iodesc, trefmxav, ierr)
          call pio_write_darray(File, trefmnav_desc, iodesc, trefmnav, ierr)
+#ifdef MAML
+         ii=0
+         tmpfld_crm(:) = fillvalue
+         do i=begchunk,endchunk
+            ncol = cam_out(i)%ncol
+            do j=1,pcols
+               do jj=1,num_inst_atm
+                  ii=ii+1
+                  if(j<=ncol) tmpfld_crm(ii) = cam_out(i)%flwds(j,jj)
+                enddo
+            end do
```

Comment:
Your spacing is inconsistent here. I would prefer that we always match the syntax style of the file we are editing and not be mixing "end do" and "enddo" right next to each other.

Reply:
I will check through the all files.